tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/PacketTracer.cs

```diff
@@ -0,0 +1,253 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using System.Runtime.Intrinsics.X86;
+using static System.Runtime.Intrinsics.X86.Avx;
+using static System.Runtime.Intrinsics.X86.Avx2;
+using static System.Runtime.Intrinsics.X86.Sse2;
+using System.Runtime.Intrinsics;
+using System;
+
+using ColorPacket256 = VectorPacket256;
+
+internal class Packet256Tracer
+{
+    public int Width { get; private set; }
+    public int Hight { get; private set; }
+    private static readonly int MaxDepth = 5;
+
+    private static readonly Vector256<float> SevenToZero = SetVector256(7f, 6f, 5f, 4f, 3f, 2f, 1f, 0f);
+
+    public Packet256Tracer(int _width, int _hight)
+    {
+        if (_width % VectorPacket256.Packet256Size != 0)
```

Comment:
Parenthesis around mathematical expressions are helpful 😄

Reply:
(not everyone may agree, but that is my preference)

Reply:
Will do.